school/static/description/index.html

```diff
@@ -83,10 +83,10 @@ <h3 class="oe_slogan"></h3>
     <div class="oe_row">
         <div class="oe_span6">
             <div class="oe_bg_img">
-                <img class="oe_picture oe_screenshot" src="school_news.png">
+                <img class="oe_picture oe_screenshot" src="news1.png">
             </div>
         </div>
-        <div class="oe_span6">
+        <div class="oe_span4">
             <p class='oe_mt32'>
                 Set latest news updates of school on the dash board automatically.
             </p>
```

Comment:
school/static/description/studentregister.png --> this picture is bad. Please put some good data, not fake like cvv fff!